Formula/code-server.rb

```diff
@@ -0,0 +1,78 @@
+class CodeServer < Formula
+  desc "Access VS Code through the browser"
+  homepage "https://coder.com"
+  # We download a built linux release but rebuild the native parts against our version of node/darwin.
+  # This just makes things easier as we don't have to rebuild all of the javascript.
+  url "https://github.com/cdr/code-server/releases/download/3.1.1/code-server-3.1.1-linux-x86_64.tar.gz"
+  version "3.1.1"
+  sha256 "5dd922d28b2e351c146081849d987fb1e439ee7d53b941434b2eecb2a194da71"
+
+  depends_on "node@12"
```

Comment:
Can it build with the latest node?

Reply:
I was under the impression we were locked onto 12 for some reason but @code-asher has clarified it should work on 13 so I'll give it a shot.